scenarios2/02_Gods.cfg

```diff
@@ -290,6 +310,30 @@
             {GOLD 1000 1200 1500}
         [/modify_side]
     [/event]
+#ifdef HARD
+    [event]
+        name=moveto
+        [filter]
+            [filter_location]
+                x,y=39,21
+                radius=20
+            [/filter_location]
+            side=2
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "There is a swamp here!"
+        [/message]
+        [message]
+            speaker=Emragiela
+            message= _ "Have I not warned you to leave me be, rabble?  And are those not elves in the distance?  You will all make perfect test subjects. Catch them, my servants!"
```

Comment:
Doubled spacing between sentences. This looks odd.

Reply:
There should be two spaces after after a period.  The odd part is I left one out after "subjects." (probably cut and paste).

I don't change this in any existing text since it would probably break translations(?), but new text or anything that changes I write correctly.

Reply:
I may have mentioned it before.  Maybe it was someone else.

Those of us who learned to type on a typewriter learned the correct usage.  Single space is now common, though it's still wrong as evidenced by the fact that (so I'm told) microsoft word considers single space to be correct.

Reply:
I did a quick check on this and I have found that what you say is true, but not for devices used to render text in present.

Two spaces after sentence used to be useful when typewriters were used, all font was in monospace and the blank area between narrow letters (such as _i_ and _l_) looked very similar to space between words, creating a need for a more pronounced separator.

But nowadays, text is rendered with different widths of letters, so there are no excessive gaps between letters, making the two spaces after dot unnecessary. Additionally, the definition of the font can take the context into consideration and have different widths for spaces that follow letters and for those that follow dots. MS word has this setting as part of the block style, I think. As a result, nearly all style guides recommend a single space after sentence.